src/search/utils/logging.cc

```diff
@@ -40,6 +40,17 @@ void add_log_options_to_feature(plugins::Feature &feature) {
         "normal");
 }
 
+
+tuple<string, Verbosity> get_log_parameters_from_options(const plugins::Options &opts){
+    auto own_tuple = make_tuple<string, Verbosity>(
+            opts.get<string>("name"), // TODO issue1082 move name into get_evaluator_parameters_from_options and rename it to *description*.
+            opts.get<Verbosity>("verbosity")
+    );
```

Comment:
Analogous comments to the above reply. `own_tuple` seems an awkward variable name.
I don't think there's a value in introducing the name just to return it, it doesn't add extra documentation.